cmd/terramate/cli/cloud.go

```diff
@@ -885,6 +992,82 @@ func (c *cli) setGitlabCIMetadata(md *cloud.DeploymentMetadata) {
 	md.GitlabCICDMergeRequestApproved = mrApproved
 }
 
+func (c *cli) setBitbucketPipelinesMetadata(md *cloud.DeploymentMetadata) {
+	md.BitbucketPipelinesBuildNumber = os.Getenv("BITBUCKET_BUILD_NUMBER")
+	md.BitbucketPipelinesPipelineUUID = os.Getenv("BITBUCKET_PIPELINE_UUID")
+	md.BitbucketPipelinesCommit = os.Getenv("BITBUCKET_COMMIT")
+	md.BitbucketPipelinesWorkspace = os.Getenv("BITBUCKET_WORKSPACE")
+	md.BitbucketPipelinesRepoSlug = os.Getenv("BITBUCKET_REPO_SLUG")
+	md.BitbucketPipelinesRepoUUID = os.Getenv("BITBUCKET_REPO_UUID")
+	md.BitbucketPipelinesRepoFullName = os.Getenv("BITBUCKET_REPO_FULL_NAME")
+	md.BitbucketPipelinesBranch = os.Getenv("BITBUCKET_BRANCH")
+	md.BitbucketPipelinesTag = os.Getenv("BITBUCKET_TAG")
+	md.BitbucketPipelinesParallelStep = os.Getenv("BITBUCKET_PARALLEL_STEP")
+	md.BitbucketPipelinesParallelStepCount = os.Getenv("BITBUCKET_PARALLEL_STEP_COUNT")
+	md.BitbucketPipelinesPRID = os.Getenv("BITBUCKET_PR_ID")
+	md.BitbucketPipelinesDestinationBranch = os.Getenv("BITBUCKET_PR_DESTINATION_BRANCH")
+	md.BitbucketPipelinesStepUUID = os.Getenv("BITBUCKET_STEP_UUID")
+	md.BitbucketPipelinesDeploymentEnvironment = os.Getenv("BITBUCKET_DEPLOYMENT_ENVIRONMENT")
+	md.BitbucketPipelinesDeploymentEnvironmentUUID = os.Getenv("BITBUCKET_DEPLOYMENT_ENVIRONMENT_UUID")
+	md.BitbucketPipelinesProjectKey = os.Getenv("BITBUCKET_PROJECT_KEY")
+	md.BitbucketPipelinesProjectUUID = os.Getenv("BITBUCKET_PROJECT_UUID")
+	md.BitbucketPipelinesStepTriggererUUID = os.Getenv("BITBUCKET_STEP_TRIGGERER_UUID")
```

Comment:
Is this the triggerer of a deployment? We can get as well the pipeline creator calling GetPipeline, no?

Reply:
this is actually the triggerer of the step. You can re-run a step in Bitbucket, if it fails. But usually yes, the step triggerer is the same as the pipeline creator but not always.

If you say fetching the pipeline creator is required to have some functionality, then yes I can fetch it.